docs/dictionary/message/currentTimeChanged.lcdoc

```diff
@@ -25,12 +25,30 @@ newTime:
 The number of the interval where the player is after the change.
 
 Description:
+The <currentTimeChanged> message is sent while the player is playing or when 
+a user changes the current time in a player.
+
 Handle the <currentTimeChanged> message if you want to do something when
 the user changes the current time in a <player>.
 
-The user can change the current time by clicking in the controller bar
-or by using the arrow keys when the player is the active (focused)
-control. 
+The user can change the current time by: 
+	- clicking the play button in the controller bar,
```

Comment:
@MartinKoob The formatting here is a bit wrong. Should be an extra line before the list and then the list shouldn't be indented:

```
The user can change the current time by: 
 +	- clicking the play button in the controller bar,
  
            
  Write  Preview

@MartinKoob The formatting here is a bit wrong. Should be an extra line before the list and then the list shouldn't be indented:
```
The user can change the current time by: 

- clicking the play button in the controller bar,
- clicking in the controller bar, 
- dragging the controller bar scroller thumb,
- using the arrow keys when the player is the active (focused)control,
- using the scrubber control in the controller bar. 
   (which is revealed by clicking the arrow buttons in the controller bar
   while holding the shift key down.)
```